syncmaster/worker/handlers/file/base.py

```diff
@@ -17,7 +19,7 @@
 
 
 class FileHandler(Handler):
```

Comment:
TODO: FileHandler is actually handler for FileDFWriter with remote FS (direct write), and FileProtocolHandler is handler for FileDFWriter with local FS (write via upload). Maybe we should keep here only common methods, like file name generator, and split other ones to classes where the method is really used.